### ISSUE TITLE
Tooltip trigger: 'item' with Showsymbol:false

### ISSUE THREAD

**Issue by chanpuraraj** [apache/echarts#14563]
### What problem does this feature solve?
Is it possible to configure tooltip:{ trigger:'item'} with showSymbol: false? Because I'm working with a stacked line graph where I want a tooltip for each area without clicking on the symbol.

Any lead would be appreciated

### What does the proposed API look like?
By doing this, one can see the tooltip in stacked line just by hovering on area

**Comment by sjcobb** [comment]
I'm also seeing this when plotting 50+ series, which requires hiding the symbol using `symbol: 'none'` or `showAllSymbol: false`. In this case, I'd like to still be able to use `trigger: 'item'` tooltips since there are so many lines. Thanks for your help, maybe there is a workaround I'm not considering?

Also, seems related to [issue 12751](https://github.com/apache/echarts/issues/12751)

**Comment by julibeg** [comment]
any update on this? Is there a workaround to use tooltips trigged with `'item'` while hiding symbols?

**Comment by yoelb00** [comment]
There is any update?
I want to show tooltip for each series separately , without showing the symbol,
Example:
https://codesandbox.io/p/sandbox/stacked-line-chart-forked-y7zn6h

**Comment by diazdressk** [comment]
> There is any update? I want to show tooltip for each series separately without showing the symbol, Example: https://codesandbox.io/p/sandbox/stacked-line-chart-forked-y7zn6h

Hello! Did you find decision?

**Comment by yoelb00** [comment]
> > There is any update? I want to show tooltip for each series separately without showing the symbol, Example: https://codesandbox.io/p/sandbox/stacked-line-chart-forked-y7zn6h
> 
> Hello! Did you find decision?

Hey, 
Nope, I'm still waiting for echarts that will respond.

**Comment by diazdressk** [comment]
> > > There is any update? I want to show tooltip for each series separately without showing the symbol, Example: https://codesandbox.io/p/sandbox/stacked-line-chart-forked-y7zn6h
> > 
> > 
> > Hello! Did you find decision?
> 
> Hey, Nope, I'm still waiting for echarts that will respond.

I found another way, try this: 

trigger: 'item',
   axisPointer: 
         {
               type: 'cross',
         },
    },

**Comment by yoelb00** [comment]
> > > > There is any update? I want to show tooltip for each series separately without showing the symbol, Example: https://codesandbox.io/p/sandbox/stacked-line-chart-forked-y7zn6h
> > > 
> > > 
> > > Hello! Did you find decision?
> > 
> > 
> > Hey, Nope, I'm still waiting for echarts that will respond.
> 
> I found another way, try this:
> 
> trigger: 'item', axisPointer: { type: 'cross', }, },

Hey,
I tried that, but when I hover over a line, it's show points on all the lines , 
I want to show the point only on the hovered line.
![Screenshot_1](https://github.com/apache/echarts/assets/46006235/dab68b75-89c6-4770-8c29-82e4342abdeb)

**Comment by diazdressk** [comment]
> > > > > There is any update? I want to show tooltip for each series separately without showing the symbol, Example: https://codesandbox.io/p/sandbox/stacked-line-chart-forked-y7zn6h
> > > > 
> > > > 
> > > > Hello! Did you find decision?
> > > 
> > > 
> > > Hey, Nope, I'm still waiting for echarts that will respond.
> > 
> > 
> > I found another way, try this:
> > trigger: 'item', axisPointer: { type: 'cross', }, },
> 
> Hey, I tried that, but when I hover over a line, it's show points on all the lines , I want to show the point only on the hovered line. ![Screenshot_1](https://private-user-images.githubusercontent.com/46006235/317723905-dab68b75-89c6-4770-8c29-82e4342abdeb.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE2MzY1NTYsIm5iZiI6MTcxMTYzNjI1NiwicGF0aCI6Ii80NjAwNjIzNS8zMTc3MjM5MDUtZGFiNjhiNzUtODljNi00NzcwLThjMjktODJlNDM0MmFiZGViLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzI4VDE0MzA1NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWNjZTgyMDNjNTg0ZDk2MzMyNzYzZDUxMTk4ZDBkZTlhMzc3NmRlNzg3YzczYTI3NGM1MmQ1ZGFjMTM5YmVhMzUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.fOC9h_GrvF7VvIZvVHJDzpz-SC-QmZDhF9M4zSS49nI)

Yep, you have pass your data to "data" field and render with formatter
![Screenshot from 2024-03-28 20-33-23](https://github.com/apache/echarts/assets/36816323/22f5540e-6fbf-4341-863b-82e9c9392fd1)
d in series
![Screenshot from 2024-03-28 20-29-08](https://github.com/apache/echarts/assets/36816323/733612ee-b402-41af-9f4c-001ed5599424)
![Screenshot from 2024-03-28 20-29-12](https://github.com/apache/echarts/assets/36816323/baf91be5-3390-4d89-9e35-8a37d6c74a21)
![Screenshot from 2024-03-28 20-29-19](https://github.com/apache/echarts/assets/36816323/1b01b3ea-12ea-4cfb-ae7f-4abca0f228a3)
![Screenshot from 2024-03-28 20-32-51](https://github.com/apache/echarts/assets/36816323/d6846f76-2ee5-4b4d-98b1-9ed88013012b)

**Comment by yoelb00** [comment]
I found a solution how to show the symbol only when hovering specific line:
I change the opacity to 0, 
and we change the opacity to 1 inside the emphasis object.
example:
https://codesandbox.io/p/sandbox/stacked-line-chart-forked-dm8kn8

**Comment by yoelb00** [comment]
> > There is any update? I want to show tooltip for each series separately without showing the symbol, Example: https://codesandbox.io/p/sandbox/stacked-line-chart-forked-y7zn6h
> 
> Hello! Did you find decision?

I found a solution how to show the symbol only when hovering specific line:
I change the opacity to 0,
and we change the opacity to 1 inside the emphasis object.
example:
https://codesandbox.io/p/sandbox/stacked-line-chart-forked-dm8kn8
